app/_features/event/components/event-stat-card.tsx

```diff
@@ -79,28 +79,42 @@ export function EventStatCard({ event }: { event: selectEvent }) {
                 {"Event is still ongoing, come back when it's finished."}
               </span>
             </div>
-          ) : stat ? (
+          ) : stat?.data ? (
             <>
               <StatList>
                 <StatItem
                   name="Registered users"
-                  value={stat.data.registeredUsers}
+                  value={
+                    stat.data.registeredUsers ? stat.data.registeredUsers : 'NA'
+                  }
```

Comment:
@GagahGK I think if the value is number, the default value should be 0 not NA (Not available).

Let's use this Registered user's stat as example. I will prefer to see:
**Registered users: 0**
than to see
**Registered users: NA**.

This is also applicable to other stat fields like Joined users, Percentage joined, etc.